index.rst

```diff
@@ -44,14 +44,639 @@
+
+The `Engineering Facilities Database (EFD) <https://sqr-034.lsst.io/#introduction>`_ records all data sent by components over the DDS control network.
+This content essentially tracks the observatory state as a function of time and is very useful in diagnosing issues and understanding (both desired and undesired) operational behaviours.
+The database is best queried using the `EFD client <https://efd-client.lsst.io/>`_ from the `Nublado (Jupyter Notebook) Interface`_ (or any python-based method/script) when making custom plots.
+Instructions on how to access the EFD, and specifically the other instances of the data, are found in `SQR-034 <https://sqr-034.lsst.io/#efd-deployments>`_.
+However, the `Chronograf`_ graphical front-end offers a nice solution for building simple plots (dashboards).
+
+The `Large File Annex (LFA) <https://tstn-029.lsst.io/>`_ contains files over ~1 MB that are accessible both from the summit and the `RSP`_.
+When a file is published to the LFA its presence (and location of the file) is published via SAL/DDS and therefore the location of LFA files can be found via an EFD query.
+It is anticipated that artifacts generated from automated on-the-fly analyses will be stored in this area.
+An example of this would be the `Papermill Executed Parameterized Notebooks`_.
+
+
+Chronograf
+''''''''''
+
+Chronograf is a tool that provides a user-friendly graphical interface to each `deployed instance of the EFD <https://sqr-034.lsst.io/#efd-deployments>`_.
+During operations, observers will use the `summit instance <https://chronograf-summit-efd.lsst.codes/>`_ (VPN required).
+It is particularly useful for quick and simple analysis as well as for creating dashboards to show a specific information about the status of the observatory; especially when a similar functionality is not available in LOVE.
+
+It is not well suited for complex queries or figures and previous queries/plots are not easily replicated.
+Furthermore, it always displays the last event seen.
+Therefore if a Controllable SAL Component (CSC) crashes, it will always show the last published state.
+For this reason (and many others), it's not an appropriate substitute for a true status GUI, such as what is being provided by the `LOVE`_ interface.
+
+Watcher
+'''''''
+
+The `Watcher CSC <https://ts-watcher.lsst.io/>`_ monitors control components listening for data that signals an alarm to the observer.
+The alarms are defined by a series of "rules" that are defined and added to the package.
+The CSC itself is not a display tool nor does it have any display functionality.
+When condition defined by an rule is met, an alarm is generated and the observer is alerted via a LOVE screen.
+The alarm has a series of levels and audible alerts are sent out via LOVE, as well as a visual notification.
+Once the alarm is acknowledged by the observer the alert is considered to be completed.
+There is no feedback or interaction for the observer beyond the acknowledgment of the alarm.
+
```

Comment:
> Alarms will only deactivate if the condition for which they are activated cease to exist, e.g., if a CSC resumes publishing heartbeats after being restored.

Reply:
tweaked a bit to say that deactivate means (stop occurring) and not just shut off.